pyro/ops/hessian.py

```diff
@@ -0,0 +1,17 @@
+import torch
+
+
+def hessian(y, xs):
+    """
```

Comment:
anyone want to add to this docstring? who wrote this? @eb8680

Reply:
`git blame` says @fehiepsi wrote `hessian()`